# Changes wget usage to _allow_ IPv6, but to prefer IPv4. This way, DNS64 and
# IPv6-only http_proxy scenarios should work.
# Patch written by Michael Mol <mikemol@gmail.com>
# Patch based on dialog with upstream, ascertaining their purpose for using
# -4, and coming up with an agreed-upon workaround. Upstream equivalent is
# committed to their vcs, and will probably appear in release 4.1.4.

*** python/lib/Variables.py	2012-07-10 20:43:56.968567114 -0400
--- python/lib/Variables-patched.py	2012-07-10 20:44:21.491898941 -0400
*************** if (os.environ["POL_OS"] == "Mac"):
*** 26,28 ****
  	os_name = "darwin"
! 	os.environ["POL_WGET"] = "wget -4 -q"
  	
--- 26,28 ----
  	os_name = "darwin"
! 	os.environ["POL_WGET"] = "wget --prefer-family=IPv4 -q"
  	
*************** if (os.environ["POL_OS"] == "Linux"):
*** 39,41 ****
  	else:
! 		os.environ["POL_WGET"] = "wget -4 -q"
  
--- 39,41 ----
  	else:
! 		os.environ["POL_WGET"] = "wget --prefer-family=IPv4 -q"
  
